### ISSUE TITLE
"Connession failed" message without mobile data or wi-fi. for example remote places

### ISSUE THREAD

**Issue by maxcalce** [gimportexportdevs/gexporter#18]
HI.
Compliments for the app.
I use it to transfer fit files from the phone's internal memory to the watch via gexporter.
If there is no mobile data connection it gives me connection failed. You can do something?
The file is already in the phone and the watch is successfully connected via bluetooth.

gpx/fit exporter has all permissions
Samsung Galaxy S21 updated
with active mobile data it works
with mobile data turned off and wifi connection it works
with mobile data disabled and wifi disabled (in the middle of a mountain without any type of network) it doesn't work and gives an error "connection failed"
I get to the point of seeing the .fit tracks saved in the phone but when downloading it gives the message.

Thank you

**Comment by haraldh** [comment]
> with mobile data disabled and wifi disabled 

Yeah, that is the "Garmin Connect" app, which does not route the requests for "makeWebRequest" if sees no network, altough the request goes to localhost (127.0.0.1) which is the phone itsself.

**Comment by haraldh** [comment]
Seems like there is a difference for `HTTP_RESPONSE_CONTENT_TYPE_JSON` for getting the tracks with:
https://github.com/gimportexportdevs/gimporter/blob/fe4711bf3fe2daa465c744195e74a7c866bb1f16/source/gimporterApp.mc#L87

and `HTTP_RESPONSE_CONTENT_TYPE_FIT` for getting the fit file with:
https://github.com/gimportexportdevs/gimporter/blob/master/source/gimporterApp.mc#L179C150-L179C180
